frontend/mockServiceWorker.js

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable */
```

Comment:
this file just gets copied into the `/storybook_public` directory when starting or building storybook